### PR TITLE
feat(Signups): CTAs for high-traffic docs

### PR DESCRIPTION
**Pull request by barbnewrelic**
For DOC-7492

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by barbnewrelic** [comment]
I'm requesting @mmfred  to review this PR because we have already been talking about how to approach the customized wording for signups CTAs.

**Review by urbiz-nr — DISMISSED** [review]
Lookin`good!

**Comment by barbnewrelic** [comment]
@urbiz-nr  Thanks for reviewing this. As I noted, this is part of a project that I am working on with @mmfred  which is why I specifically had requested her review. Really appreciate your second pair of eyes on this, though!

**Comment by barbnewrelic** [comment]
@mmfred  I am specifically requesting you to review because of the project we are working on together. No rush, I know there are a lot of docs. Thanks!

File: src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx [hunk 1]
```diff
@@ -13,13 +13,22 @@ redirects:
...
 
-For a description of synthetic monitor types, see [Types of monitor](/docs/synthetics/new-relic-synthetics/getting-started/types-synthetics-monitors).
+* Capture aggregate numbers, including an [overview](/docs/synthetics/new-relic-synthetics/dashboards/synthetics-overview-dashboard-view-monitors-performance) or [summary](/docs/synthetics/new-relic-synthetics/pages/summary-page) for ping monitors.
+* Provide detailed statistics for each [page resource](/docs/synthetics/new-relic-synthetics/dashboards/synthetics-resources-dashboard-understand-load-times), and [downtime incidents](/docs/synthetics/new-relic-synthetics/dashboards/synthetics-downtimes-dashboard-troubleshoot-downtime).
+* Collect [custom response codes](/docs/synthetics/new-relic-synthetics/using-monitors/synthetics-response-codes) for more detail on your monitor runs.
+
+You'll be able to add and configure synthetic monitors to suit your environnment after you [create a New Relic account](https://newrelic.com/signup). It's free, forever!)
```
Comment (mmfred) [inline src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx:22]:
```suggestion
You'll be able to add and configure synthetic monitors to suit your environnment after you [create a New Relic account](https://newrelic.com/signup). (It's free, forever!)
```

**Review by mmfred — APPROVED** [review]
@barbnewrelic - fantastic work on these. I really like what you've done. I appreciate the work to individualize these CTAs! Only one error that I could see.

**Pull request merged**